internal/fuzzbinder/fuzzscraper.go

```diff
@@ -302,27 +236,12 @@ func crashDetected(revisionDigest intoto.DigestSet, reader io.Reader) (*Crash, e
 
 // getGCSFileDigest gets the digest of a file stored in GCS given its name
 // and the bucket where it is stored.
-func getGCSFileDigest(bucketName string, blobName string) (*intoto.DigestSet, error) {
-	bucket, err := getBucket(bucketName)
-	if err != nil {
-		return nil, fmt.Errorf(
-			"could not get %s bucket to compute SHA256 of %s: %v", bucketName, blobName, err)
-	}
-	reader, err := getBlob(bucket, blobName)
-	if err != nil {
-		return nil, fmt.Errorf(
-			"could not get %s blob to compute SHA256: %v", blobName, err)
-	}
-	fileBytes, err := io.ReadAll(reader)
-	if err != nil {
-		return nil, fmt.Errorf(
-			"could not get data from %s reader to compute SHA256: %v", blobName, err)
-	}
+func getGCSFileDigest(fileBytes []byte) *intoto.DigestSet {
```

Comment:
We are getting less errors in the function signatures---I think that is a good sign :) !